modules/navigation/godot_navigation_server_2d.h

```diff
@@ -51,6 +51,8 @@ class GodotNavigationServer2D : public NavigationServer2D {
 	NavMeshGenerator2D *navmesh_generator_2d = nullptr;
 #endif // CLIPPER2_ENABLED
 
+	const Vector<Vector2> dummy_empty_vertices;
```

Comment:
Can be removed? I dont see any uses of this in the 2D server.

Reply:
Ah yeah, of course, it slipped my mind to remove them after removing the ref. Thanks!